src/ImageRunModal.jsx

```diff
@@ -655,6 +655,16 @@ export class ImageRunModal extends React.Component {
         if (publishValidation.some(entry => Object.keys(entry).length > 0))
             validationFailed.publish = publishValidation;
 
+        const volumesValidation = volumes.map(a => {
+            let volumesValidation;
+            volumesValidation = validateVolume(a, "hostPath", volumesValidation);
+            volumesValidation = validateVolume(a, "containerPath", volumesValidation);
```

Comment:
Same weird API, let's discuss/fix this for the port mapping. I'll ignore the other validations for the review for now.